babel.config.js

```diff
@@ -158,6 +158,7 @@ module.exports = function (api) {
 
       convertESM ? "@babel/proposal-export-namespace-from" : null,
       convertESM ? "@babel/transform-modules-commonjs" : null,
+      convertESM ? pluginNodeImportInterop : null,
```

Comment:
Q: This plugin will be no-op for babel parser and some plugins bundled by Rollup, is that intended?

Reply:
It is not a problem because all the bundled packages don't have any third-party dependency. And even if they did, it would only be a problem if these third-party dependencies used `__esModule`.

However, I asked to reopen https://github.com/rollup/plugins/issues/635 to add an option to `@rollup/plugin-commonjs` to mirror the Node.js behavior.